app/package.json

```diff
@@ -4,10 +4,17 @@
   "identifier": "com.example.electron-boilerplate",
   "description": "Starter for your Electron application. Out of the box ready for serious stuff.",
   "version": "0.1.0",
+  "build": "1",
```

Comment:
I believe you have to increment this on every build you submit for review. You may have multiple attempts to get a specific version approved.